{
  "uuid": "2d9cf3b1-6aed-42e2-9b0c-381b4bef4806",
  "path": "/lib/python3.12/lib2to3/fixes/__pycache__/fix_renames.cpython-312.pyc",
  "size": 2864,
  "modified": "2025-05-28T17:29:55.213456947Z",
  "created": "2025-05-28T17:29:55.213456947Z",
  "sha256_hash": "ba4513db4667fda58142b0229c9ffb85e72920ac2e57c6d791c1f8f043fe44ce",
  "file_type": {
    "Other": "pyc"
  },
  "permissions": 33188,
  "is_executable": false,
  "is_signed": false,
  "code_signature": null,
  "bundle_info": null,
  "scan_timestamp": "2025-06-01T18:47:09.185259065Z"
}